frontend/src/app/components/wp-card-view/wp-single-card/wp-single-card.component.sass

```diff
@@ -34,6 +31,10 @@
   wp-single-card:host.gu-transit &
     @include modifying--placeholder
 
+::ng-deep .work-packages-partitioned-query-space--container.-split
```

Comment:
I'd prefer to have this added as a global selector as `ng-deep` is deprecated

Reply:
Done @oliverguenther 👍